### PR TITLE
create profile component, add sccs file

### PR DESCRIPTION
**Pull request by Volodia1999**

### CODE REVIEW

File: client/src/components/Profile/Profile.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import '../styles/style.scss';
-import { Instagram } from 'react-content-loader';
+import {Instagram} from 'react-content-loader';
```
Comment (InnaKutsik) [inline client/src/components/Profile/Profile.tsx]: again spaces

File: client/src/components/styles/Profile.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+a {
+  text-decoration: none;
+  color: #000;
+  font-size: 1rem;
+}
+
+a:hover {
+  text-decoration: none;
+  color: black;
+}
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]: that's not scss style
where is nesting?

For example
```
a {
  text-decoration: none;
  color: #000;
  font-size: 1rem;
  
  &:hover {
      text-decoration: none;
      color: black;
  }
}
```

File: client/src/components/styles/Profile.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
+body {
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]: pls move body styles into root directory

File: client/src/components/styles/Profile.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  background-color: #f4f4f4;
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]: to variable

File: client/src/components/styles/Profile.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  font-size: 0.9rem;
+}
+
+.profile-info {
+  flex-direction: column;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]: to variable

File: client/src/components/styles/Profile.scss [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  font-size: 1rem;
+}
+
+a:hover {
+  text-decoration: none;
+  color: black;
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]: to variable

File: client/src/components/styles/Profile.scss [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+@media screen and (max-width: 383px) {
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]: to variable

File: client/src/components/styles/_variables.scss [hunk 8]
```diff
@@ -1,3 +1,25 @@
 $color-red: #f00;
 $color-grey-black: #525656;
 $color-grey: #dcdcdc;
+$color-snow: #f4f4f4;
+$color-black: #000;
+
+@mixin mobile-break() {
```
Comment (anna-kuzii) [inline client/src/components/styles/_variables.scss]: use a separate file for mixins.

File: client/src/components/Profile/Profile.tsx [hunk 9]
```diff
@@ -35,46 +38,58 @@ export default class Profile extends React.Component<IFormProps> {
...
-                            <p>
-                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Harum iusto nesciunt
-                                repudiandae vel! Accusamus aliquid animi consequatur, consequuntur cumque dolorem
-                                eum eveniet fugit iste labore magni, obcaecati perferendis rerum veniam!
-                            </p>
+                <div className='profile-header'>
+                    <div className='container'>
+                        <div className='row'>
+                            <div className='profile d-flex mt-5'>
+                                <div className='col-md-4 col-sm-4 col-4 mr-md-4 mr-sm-4'>
```
Comment (anna-kuzii) [inline client/src/components/Profile/Profile.tsx]: do we need all of these classes?

File: client/src/components/styles/_mixin.scss [hunk 10]
```diff
@@ -0,0 +1,19 @@
+@mixin mobile-break() {
+  @media screen and (max-width: 393px) {
```
Comment (anna-kuzii) [inline client/src/components/styles/_mixin.scss]: pls use breakpoints which are used everywhere and use variable for that

File: client/src/components/styles/_mixin.scss [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+    .btn {
+      font-size: 0.75rem;
+    }
+
+    .profile-name {
+      font-size: 1rem;
+    }
+
+    .followers {
+      font-size: 0.8rem;
```
Comment (anna-kuzii) [inline client/src/components/styles/_mixin.scss]: 0 is not necessary. Lint should be angry for that

File: client/src/components/styles/style.scss [hunk 12]
```diff
@@ -1,21 +1,21 @@
...
+html {
+  height: 100%;
 }
 
-.header {
-    flex: 1 0 auto;
+body {
+  min-height:100%;
+  position:relative;
+  padding-bottom: 70px;
```
Comment (anna-kuzii) [inline client/src/components/styles/style.scss]: where is space? what does lint tell u?

File: client/src/components/styles/_mixin.scss (hunk 11 above)
Comment (Volodia1999) [reply client/src/components/styles/_mixin.scss]: Lint is not angry)

File: client/src/components/styles/Profile.scss (hunk 2 above)
Comment (Volodia1999) [reply client/src/components/styles/Profile.scss]: Fixed

File: client/src/components/styles/Profile.scss (hunk 3 above)
Comment (Volodia1999) [reply client/src/components/styles/Profile.scss]: Fixed

File: client/src/components/styles/Profile.scss (hunk 4 above)
Comment (Volodia1999) [reply client/src/components/styles/Profile.scss]: Fixed

File: client/src/components/styles/Profile.scss (hunk 5 above)
Comment (Volodia1999) [reply client/src/components/styles/Profile.scss]: Fixed

File: client/src/components/styles/Profile.scss (hunk 6 above)
Comment (Volodia1999) [reply client/src/components/styles/Profile.scss]: Fixed

File: client/src/components/styles/Profile.scss (hunk 7 above)
Comment (Volodia1999) [reply client/src/components/styles/Profile.scss]: Fixed

File: client/src/components/styles/_variables.scss (hunk 8 above)
Comment (Volodia1999) [reply client/src/components/styles/_variables.scss]: Fixed

File: client/src/components/Profile/Profile.tsx (hunk 9 above)
Comment (Volodia1999) [reply client/src/components/Profile/Profile.tsx]: Fixed, delete unnecessary classes

File: client/src/components/styles/_mixin.scss (hunk 10 above)
Comment (Volodia1999) [reply client/src/components/styles/_mixin.scss]: Fixed

File: client/src/components/styles/style.scss (hunk 12 above)
Comment (Volodia1999) [reply client/src/components/styles/style.scss]: Fixed

File: client/src/components/Profile/Profile.tsx [hunk 13]
```diff
@@ -35,46 +38,56 @@ export default class Profile extends React.Component<IFormProps> {
...
     public componentWillUnmount(): void {
         clearTimeout(this.timerHandle);
         this.timerHandle = 0;
     }
 
     public render(): JSX.Element {
-        const { user: {first_name, last_name, avatar} }: any = this.props;
-        const { loaded }: {loaded: boolean} = this.state;
+        const {user: {first_name, last_name, avatar}}: any = this.props;
+        const {loaded}: { loaded: boolean } = this.state;
```
Comment (InnaKutsik) [inline client/src/components/Profile/Profile.tsx]: again spaces

File: client/src/components/styles/Profile.scss [hunk 14]
```diff
@@ -0,0 +1,29 @@
+@import "variables";
+@import "mixin";
+
+.profile {
+  margin: 0 auto;
+  @include mobile-break;
```
Comment (InnaKutsik) [inline client/src/components/styles/Profile.scss]:
```
.profile {
  margin: 0 auto;
  @include mobile-break;
  
  &-name {
      font-weight: 700;
      font-size: 1.3rem;
    }

  &-info {
      flex-direction: column;
    }
}

**Pull request merged**